Source/Diagnostics/WarpXOpenPMD.H

```diff
@@ -152,6 +152,15 @@ public:
   /** Return OpenPMD File type ("bp5", "bp4", "h5" or "json")*/
   std::string OpenPMDFileType () { return m_OpenPMDFileType; }
 
+  /** Flush a few BTD buffers in a snapshot
+   *
+   * This function is controlled by the parameter
+   * FlushFormatOpenPMD::m_NumAggBTDBufferToFlush (default to 5),
+   * it can be adjusted in the input file: diag_name.buffer_flush_limit_btd
+   *
+   * @param[in] isBTD if the current diagnostic is BTD
+   */
+  void ForceFlush (bool isBTD);
```

Comment:
```suggestion
  /** Ensure BTD buffers are written to disk
   *
   * This function can be called to intermediately ensure ADIOS buffered "steps"
   * are written to disk, including valid metadata. This is needed to read partial
   * data while a simulation is running or to support restarting (the BTD diagnostics)
   * in WarpX, so it can continue to append to a partially written labframe station
   * after restart.
   */
  void FlushBTDtoDisk ();
```

Reply:
added